bemani/api/objects/catalog.py

```diff
@@ -183,7 +181,7 @@ def fetch_v1(self, idtype: str, ids: List[str], params: Dict[str, Any]) -> Dict[
 
         # Fetch the songs
         songs = self.data.local.music.get_all_songs(self.game, self.music_version)
-        if self.game == GameConstants.JUBEAT and self.version == VersionConstants.JUBEAT_CLAN:
+        if self.game == GameConstants.JUBEAT and self.version == VersionConstants.JUBEAT_CLAN or self.version == VersionConstants.JUBEAT_FESTO:
```

Comment:
Needs parens for order of operations, not sure what the intent is here. Does Festo also have the same song range?

Reply:
The answer to this would be yes. The exact same issue clan has exists in festo as well

Reply:
Okay, we should parenthesize the jubeat version portion then, or specify `self.version in (CLAN/FESTO) for correctness.

```
>>> False and False or True
True
>>> False and (False or True)
False
```